src/kitconcept/seo/behaviors/seo.py

```diff
@@ -34,6 +45,37 @@ class ISeo(model.Schema):
         max_length=155,
     )
 
+    # https://support.google.com/webmasters/answer/93710?hl=en
+    seo_noindex = schema.Bool(
+        title=_(u"No Index"),
+        description=_(u"Prevents a page from appearing in search engines"),
+        required=False,
+    )
+
+    # https://support.google.com/webmasters/answer/96569?hl=en
+    seo_nofollow = schema.Bool(
```

Comment:
Not sure if it makes sense to set this on page level. This should be an attribute of individual links. Google now interprets three values "nofollow,ugc,sponsored".